modules/ROOT/pages/appendix/gql-conformance/index.adoc

```diff
@@ -1,8 +1,8 @@
 :description: Overview of Cypher's conformance to GQL.
 = GQL conformance
 
-*Last updated*: 24 October 2024 +
-*Neo4j version*: 5.25
+*Last updated*: 23 January October 2025 +
+*Neo4j version*: 2025.02
```

Comment:
I guess this will be updated before merge :)

Reply:
Ah, I missed that - yep.